concent_api/core/payments/backends/mock.py

```diff
@@ -36,3 +38,40 @@ def get_deposit_value(client_eth_address: str) -> int:  # pylint: disable=unused
         return 0
     else:
         return 10000
+
+
+def force_subtask_payment(
+    requestor_eth_address: str,  # pylint: disable=unused-argument
+    provider_eth_address: str,  # pylint: disable=unused-argument
+    value: int,  # pylint: disable=unused-argument
+    subtask_id: str,  # pylint: disable=unused-argument
+) -> Transaction:
+    return Transaction(
+        nonce=1,
+        gasprice=10 ** 6,
+        startgas=80000,
+        to=b'7917bc33eea648809c28',
+        value=10,
+        v=1,
+        r=11,
+        s=12,
+        data=b'3078666333323333396130303030303030303030303030303030303030303030303032393662363963383738613734393865663463343531363436353231336564663834666334623330303030303030303030303030303030303030303030303030333431336437363161356332633362656130373531373064333239363566636161386261303533633030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303237313030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303562343336643666',
+    )
+
+
+def cover_additional_verification_cost(
+    client_eth_address: str,  # pylint: disable=unused-argument
+    value: int,  # pylint: disable=unused-argument
+    subtask_id: str,  # pylint: disable=unused-argument
+) -> Transaction:
+    return Transaction(
```

Comment:
it's the same transaction as in `force_subtask_payment`, so maybe it can be extracted to a constant, i.e.: `MOCK_TRANSACTION` or sth similar

Reply:
:+1: done